common/rpctypes/jsonrpc.go

```diff
@@ -39,5 +39,8 @@ type Error struct {
 
 // Error ...
 func (e *Error) Error() string {
-	return fmt.Sprintf("message=%s; code=%d; data=%v", e.Message, e.ErrorCode, e.Data)
+	if e.ErrorCode != 0 {
+		return fmt.Sprintf("message=%s; code=%d; data=%v", e.Message, e.ErrorCode, e.Data)
+	}
+	return e.Message
```

Comment:
In this PR we are returning information important to the end user in the error returned from the server, so I'm trying to remove unhelpful wrapped verbiage that makes it harder for the end user to see the helpful portion of the message.